web/pgadmin/browser/server_groups/servers/databases/schemas/tables/constraints/index_constraint/static/js/unique_constraint.ui.js

```diff
@@ -256,6 +257,17 @@ export default class UniqueConstraintSchema extends BaseUISchema {
           return {condeferred: false};
         }
       }
+    },{
+      id: 'indnullsnotdistinct', label: gettext('NULLs not distinct?'),
+      type: 'switch', group: gettext('Definition'),
+      readonly: function(state) {
+        return obj.isReadOnly(state);
+      },
+      visible: function() {
```

Comment:
Use `min_version` key instead of custom logic.

Reply:
This logic is already being used at other places.

Reply:
Please review now.

Reply:
@pravesh-sharma then remove at other places also.

Reply:
Changes done